.github/scripts/merge-benchmarks.py

```
@@ -0,0 +1,259 @@
+        target_id = target_meta.get("id")
+        if not target_id:
+            continue
+        register_target(target_id, target_meta)
+
+key = lambda row: (
+    row.get("commit_sha"),
+    row.get("target"),
+    row.get("metric"),
+    row.get("key"),
+    row.get("generated_at"),
+)
```

Ruff (0.15.9)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `key` as a `def`

(E731)